src/docs/patterns/navigation.mdx

```diff
@@ -1,70 +1,220 @@
+<ImageGrid images={[NavigationPermanentImg]} regularWidth />
 
-### Content-Resize / Content-Push
+## Rail
 
-<ImageGrid images={[PermanentImg]} />
+A navigation rail is a permanently collapsed drawer. The rail is able to show less information than a standard drawer, but has the advantage of a more compact, space-saving package. Use this variation if you have:
 
-The content-resize implementation does not cover any of the main application content. Rather, the navigation panel slides in and the main application container resizes to accommodate (the navigation panel and main panel are side by side instead of one on top of the other).
+-   fewer than seven top-level pages
+-   a flat navigation hierarchy
+-   limited space - recommended for HMI devices
 
-### Content-Cover
+<ImageGrid images={[NavigationRailImg]} regularWidth />
 
-<ImageGrid images={[TemporaryImg]} />
+## Persistent
 
-<!-- @TODO switch side-navigation links over to the "Component Recommendations style" instead of a "Interactive Code example" once we have one designed. -->
+A persistent drawer is an always-visible drawer that can be opened and closed (expanded or collapsed). In its opened state, it looks and behaves like a permanent drawer. It the closed state, it looks and behaves like a navigation rail. The drawer is typically opened and closed by clicking an icon in the header. It can also be opened momentarily on desktop applications by hovering with the mouse. Use this variation if you have:
 
-<DemoCard
-    repository={'side-navigation'}
-    angular={'https://pxblue-components.github.io/angular/?path=/story/components-drawer--with-full-config'}
-    react
-/>
+-   fewer than seven top-level pages
+-   primary pages with labels that require complex icons or may be too long to wrap effectivly
+-   limited space but require a nested navigation hierarchy
+
+<React.Fragment>
+    <ImageGrid
+        images={[CollapsibleImg]}
+        caption={[
+            'Figure 1 displays the normal collapsed state of the menu.',
+            'Figure 2 displays the open state of the menu.',
+        ]}
+        regularWidth
+    />
+    <ImageGrid
+        images={[PersistentImg, PermanentImg]}
+        caption={
+            'When the user hovers on the Navigation Rail, the side navigation will expand temporarily, revealing the text details of each navigation item; when the user moves their cursor away, the whole side navigation panel collapses.'
+        }
+    />
+</React.Fragment>
+
+## Temporary
+
+A temporary drawer is invisible until it is opened — typically by pressing a button in the [App Bar](/patterns/appbar). This variation of the drawer appears on top of the main navigation, typically using a dark overlay to block interaction with the main content. Use this variation if you have:
+
+-   a mobile application
+-   extreme space limitations
+
+<React.Fragment>
+    <ImageGrid images={[NavigationTemporaryImg]} regularWidth />
```

Comment:
![image](https://user-images.githubusercontent.com/8997218/94962356-515bbd80-04c4-11eb-9fa8-c1de5f22763c.png)

This is not a temporary drawer

Reply:
then what is

Reply:
A temporary drawer always hovers above the content and usually accompanies a backdrop